[2016-06-24 00:06:13] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:06:38] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:07:19] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:07:57] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:08:30] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:08:44] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:09:21] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:09:43] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:10:24] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:11:33] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:11:46] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:13:48] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:16:12] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:16:22] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:20:06] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:20:17] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:20:22] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:21:38] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:22:25] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:22:37] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:23:02] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:30:33] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:30:39] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-24 00:31:01] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
